src/MooLite/plugins/WhisperManager/WhisperManagerPluginDisplay.vue

```diff
@@ -0,0 +1,198 @@
+        <div
+            v-if="!Object.keys(conversations).length"
+            class="flex flex-col justify-center gap-y-3 items-center w-full h-full text-center"
+        >
+            <span class="bold text-4xl">No whispers!</span>
+            <span class="text-xl">
+                Start one by clicking a users name in chat and selecting the whisper option. You can also type the
+                following in chat:
+            </span>
+            <code class="text-xl p-1.5 rounded-mwi-default bg-midnight-300">/w &lt;username&gt;</code>
+        </div>
+        <div v-else class="flex flex-col space-y-1.5 h-full max-h-full mb-1.5">
+            <button
+                v-if="hiddenConversationCount"
+                class="bg-midnight-500 font-semibold p-1.5 rounded-mwi-default text-dark-mode hover:bg-midnight-300"
+                @click="unhideAllConversations()"
+            >
+                Unhide All
+            </button>
+            <input
+                v-if="visibleConversationCount > 4"
+                v-model="searchText"
+                type="text"
+                placeholder="Search users..."
+                class="p-1 bg-midnight-500 rounded-mwi-default my-2 text-dark-mode w-full"
+            />
+            <div class="flex flex-row space-x-0.5 overflow-y-visible overflow-x-auto">
+                <div
+                    v-for="[user, conversation] in tabList"
+                    class="flex-grow flex flex-row align-middle items-center rounded-t-mwi-default py-1 px-2 cursor-pointer relative overflow-visible"
+                    :class="{
+                        'bg-space-800 justify-between gap-2': activeConversationName === user,
+                        'justify-center': !(activeConversationName === user),
+                        'bg-midnight-500': !(activeConversationName === user) && !conversations[user].unread,
+                        'bg-midnight-400': conversations[user].unread,
+                    }"
+                    @click="onTabClick(user)"
+                >
+                    <div v-if="conversations[user].unread" class="absolute top-0 right-0">
+                        <div class="bg-warning rounded-full w-2 h-2"></div>
+                        <div class="bg-warning rounded-full w-2 h-2 animate-ping absolute top-0 right-0"></div>
+                    </div>
+                    <span class="font-semibold text-base text-dark-mode">
+                        {{ user }}
+                    </span>
+                    <CrossIcon
+                        v-if="activeConversationName === user"
+                        class="h-5 aspect-square fill-dark-mode stroke-3 rounded-full p-1 transition-colors hover:fill-progress hover:bg-neutral-200"
+                        @click="conversation.hide()"
+                    />
+                </div>
+            </div>
+            <!-- Have to add this as a style tag or it gets overridden. -->
+            <MooDivider style="margin-top: 0" />
```

Comment:
Does the `mt-0` class not work?

Reply:
No, it is overridden.

Reply:
Also I plan to build a full-fledged tab widget in the design system, so this will be replaced in the future.

Reply:
:+1: